tasks/libs/common/docker_hub.py

```diff
@@ -0,0 +1,16 @@
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry  # type: ignore
+
+
+def get_latest_version(repository, namespace="datadog") -> str:
```

Comment:
I’m not a python specialist. So, I apologize in advance if my question is naive. ;-)
I know that typing in python is optional but why putting a type annotation for the return value but not for the input parameters ?

```suggestion
def get_latest_version(repository: str, namespace: str = "datadog") -> str:
```

Reply:
good catch!